### ISSUE TITLE
[BUG] UMAPINFO Branch Lump name too long error bombs out wad load.

### ISSUE THREAD

**Issue by rakohus** [odamex/odamex#412]
**Describe the bug**
Lump name too long error bombs out wad load.

**Build that the bug occurred in**
bug/umapinfo-fixes cb6e1ca

**To Reproduce**
Load 32in24-17.wad

**Expected behavior**
Wad should load

**Additional Info**
Tagging @DeathEgg

**Screenshots, NetDemos, & Crash Dumps**
```[00:15:18] Could not load new resource files.
Lump name "$LUNCHABL" too long. Maximum size is 8 characters.
Reloading previous resource set...``

**Comment by leximayfield** [comment]
This is due to the fact that this WAD's ZMAPINFO uses an [intermission script](https://zdoom.org/wiki/Intermission_script).  Lumps containing intermission scripts are prefixed with a $, so this causes the lump lookup to fail, and even if it succeeds it's not like we can do anything with it.

**Comment by rakohus** [comment]
Are we unable to ignore lines or entries that Odamex doesn't understand? The wad used to load prior to the new mapinfo parsing

**Comment by rakohus** [comment]
Confirmed fixed in 2702beb. Thanks @AlexMax
